front-end/src/app/shared/job-card/job-card.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnInit } from "@angular/core";
+
+import { isCompanyWithLogo } from "../../model/company.model";
+import { Position } from "../../model/job.model";
+
+@Component({
+  selector: "app-job-card",
+  standalone: true,
+  imports: [],
+  templateUrl: "./job-card.component.html",
+  styleUrl: "./job-card.component.scss",
+})
+export class JobCardComponent implements OnInit {
+  @Input({ required: true })
+  public position?: Position;
+
+  ngOnInit() {
+    if (!this.position) {
+      throw new Error("Position must be defined");
+    }
+  }
+
+  protected readonly isCompanyWithLogo = isCompanyWithLogo;
```

Comment:
What is this?

Reply:
It seems like when I'm using a method in the html, it can't directly use an exported method from outside the class.